rest-server/src/controllers/job.js

```diff
@@ -16,87 +16,53 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 // module dependencies
+const httpStatus = require('http-status');
 const Job = require('../models/job');
-const logger = require('../config/logger');
 
 /**
  * Load job and append to req.
  */
 const load = (req, res, next, jobName) => {
   new Job(jobName, (job, error) => {
     if (error) {
-      if (error.message === 'JobNotFound') {
-        if (req.method !== 'PUT') {
-          logger.warn('load job %s error, could not find job', jobName);
-          return res.status(404).json({
-            error: 'JobNotFound',
-            message: `could not find job ${jobName}`,
-          });
+      if (error.message === 'Could not find job.') {
```

Comment:
>message [](start = 16, length = 7)

Be sure to use "error" instead of "message" anywhere to ensure we work on structured data instead of unstructured human read text.